src/bin/bpf-linker.rs

```diff
@@ -216,7 +218,16 @@ fn main() -> anyhow::Result<()> {
         export,
         fatal_errors,
         _debug,
-    } = Parser::try_parse_from(args)?;
+    } = match Parser::try_parse_from(args) {
+        Ok(command_line) => command_line,
+        Err(err) => match err.kind() {
+            ErrorKind::DisplayHelp | ErrorKind::DisplayVersion => {
+                print!("{err}");
+                return Ok(());
```

Comment:
If an error were returned as is, it would display as:
```
Error: bpf-linker 0.9.12
```

Therefore, I made sure to return `Ok` instead.